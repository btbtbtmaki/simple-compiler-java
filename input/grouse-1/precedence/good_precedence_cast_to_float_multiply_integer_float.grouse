main {

  print 55 :float * 9.5 nl;
  print 99 :float * 5.5 nl;
  print 71 :float * 2.7 nl;
  print 44 :float * 7.9 nl;
  print 84 :float * 8.8 nl;
  print 2 :float * 9.0 nl;
  print 66 :float * 3.9 nl;
  print 27 :float * 9.1 nl;
  print 40 :float * 4.2 nl;
  print 61 :float * 5.2 nl;
  print 93 :float * 4.6 nl;
  print 71 :float * 2.5 nl;

  print 3.2 * 76 :float nl;
  print 2.9 * 13 :float nl;
  print 7.4 * 38 :float nl;
  print 9.7 * 57 :float nl;
  print 9.9 * 79 :float nl;
  print 9.6 * 26 :float nl;
  print 5.8 * 15 :float nl;
  print 2.1 * 85 :float nl;
  print 5.8 * 19 :float nl;
  print 3.4 * 91 :float nl;
  print 4.8 * 72 :float nl;
  print 4.7 * 85 :float nl;

  print 41 :float * 96 :float nl;
  print 87 :float * 26 :float nl;
  print 32 :float * 57 :float nl;
  print 41 :float * 40 :float nl;
  print 34 :float * 85 :float nl;
  print 53 :float * 7 :float nl;
  print 1 :float * 24 :float nl;
  print 27 :float * 55 :float nl;
  print 43 :float * 73 :float nl;
  print 72 :float * 97 :float nl;
  print 98 :float * 39 :float nl;
  print 41 :float * 94 :float nl;

}

