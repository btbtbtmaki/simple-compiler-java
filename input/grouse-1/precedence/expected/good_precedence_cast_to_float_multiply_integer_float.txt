522.5
544.5
191.7
347.6
739.2
18
257.4
245.7
168
317.2
427.8
177.5
243.2
37.7
281.2
552.9
782.1
249.6
87
178.5
110.2
309.4
345.6
399.5
3936
2262
1824
1640
2890
371
24
1485
3139
6984
3822
3854
